stigs/system/ksp-audit-cassandra-stig-v-72671.yaml

```diff
@@ -0,0 +1,25 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+# Reference: https://www.stigviewer.com/stig/vrealize_-_cassandra/2017-06-06/finding/V-72671
+# Suggested Fix: chown admin /usr/lib/vmware-vcops/user/conf/cassandra/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-cassandra-stig-v-72671
+  namespace: default # Change your name space
+spec:
+  tags: ["STIGS", "CASSANDRA", "STIG V-72671", "database"]
+  message: "Alert!  file has been accessed"
+  selector:
+    matchLabels:
+      app: cass-server # Change your matchLabels
+  file:
+    severity: 3
+    matchDirectories:
+    - dir: /usr/lib/vmware-vcops/user/conf/cassandra/ 
+      readOnly: true
```

Comment:
@rohitrishim based on the rule, the read-only parameter is not required for this policy. 
cc: @raviknox @salman-accuknox @syeddhadi